packages/inscription/src/inscription/InscriptionView.tsx

```diff
@@ -30,7 +30,7 @@ const InscriptionView = ({ pid, ...props }: ComponentProps<typeof App>) => {
   });
   return (
     <div style={{ width: width }}>
-      <div className={`inscription-resizer${resizeActive ? ' active' : ''}`} {...moveProps} />
+      <div tabIndex={0} className={`inscription-resizer${resizeActive ? ' active' : ''}`} {...moveProps} />
```

Comment:
Is there a visual feedback when the resizer is focused?

Reply:
jup 
![image](https://github.com/user-attachments/assets/84f9ad5c-b6c6-4b9b-a892-a517303fad8d)